src/test/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreDocumentAdapterTest.java

```diff
@@ -262,39 +381,35 @@ public void getDocumentField() throws DatabaseAccessException, DocumentException
                 willReturn(aString).given(documentSnapshot).get(anyString());
 
                 Object result = adapter.getDocumentField(documentPath, field);
-                assertEquals(aString, result,
-                    "Should return " + aString + " as the value of the " + field + " in the " + "document located at "
-                        + documentPath);
+                assertEquals(aString, result, "Should return " + aString + " as the value of the " + field + " in the "
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String " in the " appears 8 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5465948)